src/main/channel/send-to-agnostic-channel.injectable.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { lensWindowInjectionToken } from "../start-main-application/lens-window/application-window/lens-window-injection-token";
+import { pipeline } from "@ogre-tools/fp";
+import { getInjectable } from "@ogre-tools/injectable";
+import { filter } from "lodash/fp";
+import { sendToAgnosticChannelInjectionToken } from "../../common/sync-box/channel/send-to-agnostic-channel-injection-token";
+
+const sendToAgnosticChannelInjectable = getInjectable({
+  id: "send-to-agnostic-channel-main",
+
+  instantiate: (di) => {
+    const getAllLensWindows = () => di.injectMany(lensWindowInjectionToken);
+
+    return (channel, message) => {
+      const visibleWindows = pipeline(
+        getAllLensWindows(),
+        filter((lensWindow) => !!lensWindow.visible),
+      );
+
+      visibleWindows.forEach((lensWindow) =>
+        lensWindow.send({ channel: channel.id, data: [message] }),
```

Comment:
Won't this throw errors if the splash window is still active?

Reply:
This abstraction should work in a way that it sends to all visible windows, and it doesn't matter whether it's splash or application window. However, in the code that actually does the sending using `IPC` there should be mechanism that turns the `throw` to something less hostile. 

Need to verify if that's already in place since that's existing code.

Reply:
Verified that sending to window without listener doesn't throw any errors.